packages/commonwealth/server/migrations/20240605094454-community-profile-counts.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        'Communities',
+        'profile_count',
+        {
+          type: Sequelize.INTEGER,
+          allowNull: false,
+          defaultValue: 0,
+        },
+        { transaction: t },
+      );
+      await queryInterface.addColumn(
+        'Communities',
+        'count_updated',
+        {
+          type: Sequelize.BOOLEAN,
+          allowNull: false,
+          defaultValue: false,
+        },
+        { transaction: t },
```

Comment:
Making a boolean flag into a nullable date provides a bit more flexibility, particularly in cases where we want to track if data is stale or not.

Reply:
Probably not, there is no good way to hold the transaction here as it will lock the tables. This script is meant to be run async anyways and can be run multiple times, so there should be no data consistency issues.

Reply:
So for context I am using the expand-contract pattern here https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern.

This PR represents the expand portion. This flag only exists to keep track of what counts have been successfully migrated. There will be no stale data since data can be divided into one of 2 categories:
1. pre-existing data that we need to migrate, flag starts as false, after migration set to true
2. new communities that will be added after this PR, the count_updated gets set to true and is not needed to be fixed by the async migration script.

So this flag will only be here for part 1 of this PR. part 2 will remove this flag and the old address_count column.

Reply:
Makes sense, thanks for explaining.